crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -257,8 +260,8 @@ void LateUpdateScale()
             // reach maximum detail at slightly below sea level. this should combat cases where visual range can be lost
             // when water height is low and camera is suspended in air. i tried a scheme where it was based on difference
             // to water height but this does help with the problem of horizontal range getting limited at bad times.
-            float maxDetailY = SeaLevel - _maxVertDispFromWaves / 5f;
-            float camDistance = Mathf.Abs(_viewpoint.position.y /*- maxDetailY*/);
+            float maxDetailY = SeaLevel - _maxVertDispFromWaves * _dropDetailHeightBasedOnWaves;
```

Comment:
```C#
float lowerBoundY = SeaLevel - _maxVertDispFromWaves * _maxDetailRangeWaveHeightMultiplier;
float upperBoundY = SeaLevel + _maxVertDispFromWaves * _maxDetailRangeWaveHeightMultiplier;
float camDistance = Mathf.Abs(
    Mathf.Max(
        Mathf.Min(_viewpoint.position.y - lowerBoundY, _viewpoint.position.y - upperBoundY),
        0
    )
);
```

Could having a maximum detail range multiplier like the this be a better solution to the above? (Or having them independently control an upper and lower bound?)